### PR TITLE
deps: update to version 8 of the theme

### PR DESCRIPTION
**Pull request by jashlu**
Taking out some of the outdated stuff made it run for me. I also tried with `image: node:latest` in the `docker-compose.yml` and that worked too. But i switched back to 18 and it worked as well

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ services:
       - .:/app:delegated
     ports:
       - "8000:8000"
-    command: npm run develop --verbose -- -H 0.0.0.0
```
Comment (hollandjg) [inline docker-compose.yml:10]: Is removing the `--verbose` here necessary? It might be important on the Pages.gov side for their logging.

**Review by hollandjg — APPROVED** [review]
I found the same things when trying to work out the SiteTitle, so this looks great. One clarifying question below.

- I've got a couple of additions in #83 which we can either merge in here or add as a separate PR. Your call, Josh!

**Comment by jashlu** [comment]
> I found the same things when trying to work out the SiteTitle, so this looks great. One clarifying question below.
> 
> * I've got a couple of additions in [deps: add package-lock.json, delete unused SiteTitle shadowed component #83](https://github.com/thepolicylab-projectportals/fed-content/pull/83) which we can either merge in here or add as a separate PR. Your call, Josh!

Feel free to merge in! Thanks John

File: docker-compose.yml (hunk 1 above)
Comment (jashlu) [reply docker-compose.yml:10]: Oh thanks for noticing that, I had copy/pasted the original cloud pages example again over to start fresh, I'll add that in

**Pull request merged**